File: Front/ProEvents-App/src/app/models/Speaker.ts
import { EventSpeaker } from "./EventSpeaker";
import { SocialMedia } from "./SocialMedia";

export interface Speaker {
  id: number;
  name: string;
  resume: string;
  imageURL: string;
  phone: string;
  email: string;
  socialMedias: SocialMedia;
  eventSpeakers: EventSpeaker;
}

File: Back/src/ProEvents.Domain/Speaker.cs
using System.Collections.Generic;
using ProEvents.Domain.Identity;

namespace ProEvents.Domain
{
    public class Speaker
    {
        public int Id { get; set; }
        public string Resume { get; set; }
        public IEnumerable<SocialMedia> SocialMedias { get; set; }
        public IEnumerable<EventSpeaker> EventSpeakers { get; set; }
        public User User { get; set; }
        public int UserId { get; set; }
    }
}